test/e2e/common/node/helpers/resize.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package pod
+package helpers
```

Comment:
any chance to have a more descriptive package name than "helpers"? Perhaps splitting this package in more subpackages in the same hierarchy?
I would really like to avoid this kind of names, but I guess my comment is more a (strong) wish than a requirement
(rationale: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common)

Reply:
Sure. I'm all for smaller packages.

Reply:
Moved to `test/e2e/common/node/framework/podresize`

Reply:
Sorry, looks like I forgot to push the changes.